contact/pubspec.yaml

```diff
@@ -18,7 +18,7 @@ dependencies:
   jmap_dart_client:
     git:
       url: https://github.com/linagora/jmap-dart-client.git
-      ref: main
+      ref: bugfix/save-identity-for-draft
```

Comment:
Please bump version `jmap_dart_client` to use main branch

Reply:
Done